lib/utils/__tests__/isHexColor.test.mjs

```diff
@@ -0,0 +1,13 @@
+import isHexColor from '../isHexColor.js';
+import valueParser from 'postcss-value-parser';
+
+test('isHexColor', () => {
+	expect(isHexColor(valueParser('#fff').nodes[0])).toBe(true);
+	expect(isHexColor(valueParser('#abcabc').nodes[0])).toBe(true);
+	expect(isHexColor(valueParser('#f1f2f3').nodes[0])).toBe(true);
+	expect(isHexColor(valueParser('#123').nodes[0])).toBe(true);
+	expect(isHexColor(valueParser('#123123').nodes[0])).toBe(true);
+	expect(isHexColor(valueParser('ffffff').nodes[0])).toBe(false);
+	expect(isHexColor(valueParser('#-111').nodes[0])).toBe(false);
+	expect(isHexColor(valueParser(' #fff').nodes[0])).toBe(false);
+});
```

Comment:
[suggest] Minor. How about using [`test.each()`](https://jestjs.io/docs/api#testeachtablename-fn-timeout) for more simplicity?

```js
test.each([
  ['#fff', true],
  // other cases...
])('isHexColor(%s)', (argument, expected) => {
	expect(isHexColor(valueParser(argument).nodes[0])).toBe(expected);
});
```